### PR TITLE
Add donation page

### PR DESCRIPTION
**Pull request by ubruhin**
To give a bit more information than just the two buttons in the banner.

### CODE REVIEW

File: content/donate/index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: Donate to LibrePCB
+type: donate
+---
+
+LibrePCB is an open source project – it's made by the community, for the
+community. This means that everyone can use LibrePCB without paying for it.
+
+But if you like LibrePCB, please consider to support the development of LibrePCB
```
Comment (dbrgn) [inline content/donate/index.md]:
```suggestion
But if you like LibrePCB, please consider suporting the ongoing development of LibrePCB
```

Comment (ubruhin) [reply content/donate/index.md]: Haha, a typo in the suggestion^^

Comment (dbrgn) [reply content/donate/index.md]: Haha :D

File: content/donate/index.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+title: Donate to LibrePCB
+type: donate
+---
+
+LibrePCB is an open source project – it's made by the community, for the
+community. This means that everyone can use LibrePCB without paying for it.
+
+But if you like LibrePCB, please consider supporting the ongoing development of
```
Comment (rnestler) [inline content/donate/index.md]: I'd drop the "But".

Comment (rnestler) [reply content/donate/index.md]: Heh and I'm only 4 days late, whats wrong with me :laughing:

Comment (ubruhin) [reply content/donate/index.md]: Yep, too late ;)

**Pull request merged**